### PR TITLE
feat: Add helper contract to manage risk params of GHO reserve

### PR DESCRIPTION
**Pull request by foodaka**
Adds the gho manager contract which has the responsibility to update the gho discount strategy, discount lock period, and interest rate strategy for gho.

closes https://github.com/aave/gho-core/issues/292

### CODE REVIEW

File: src/contracts/facilitators/aave/misc/GhoManager.sol [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * @notice Helper contract for managing key risk parameters of the GHO reserve within the Aave Facilitator
+ * @dev This contract is intended to be granted as PoolAdmin
+ */
+contract GhoManager is Ownable {
+  /**
+   * @dev Constructor.
+   * @param owner The owner address of this contract
+   */
+  constructor(address owner) {
+    transferOwnership(owner);
```
Comment (miguelmtzinf) [inline src/contracts/facilitators/aave/misc/GhoManager.sol:20]: No need to transfer the owner at deployment time. I d remove the whole constructor

Comment (foodaka) [reply src/contracts/facilitators/aave/misc/GhoManager.sol]: updated in `89c8f02f33d6deca6392c879a01a8976940d1376`

**Review by miguelmtzinf — APPROVED** [review]

File: deploy/10_deploy_ghomanager.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const func: DeployFunction = async function ({
+  getNamedAccounts,
+  deployments,
+}: HardhatRuntimeEnvironment) {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const addressesProvider = await getPoolAddressesProvider();
+
+  const ghoManager = await deploy('GhoManager', {
```
Comment (miguelmtzinf) [inline deploy/10_deploy_ghomanager.ts:14]:
```suggestion
const func: DeployFunction = async function ({
  getNamedAccounts,
  deployments,
}: HardhatRuntimeEnvironment) {
  const { deploy } = deployments;
  const { deployer } = await getNamedAccounts();

  const ghoManager = await deploy('GhoManager', {
```

File: deploy/10_deploy_ghomanager.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { getPoolAddressesProvider } from '@aave/deploy-v3';
+
+const func: DeployFunction = async function ({
+  getNamedAccounts,
+  deployments,
+}: HardhatRuntimeEnvironment) {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const addressesProvider = await getPoolAddressesProvider();
```
Comment (Zer0dot) [inline deploy/10_deploy_ghomanager.ts:12]:
```suggestion
```

Think this is unneeded!

**Review by Zer0dot — APPROVED** [review]

**Pull request merged**